raiden-dapp/src/components/ConnectDialog.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <raiden-dialog
+    class="connect"
+    :visible="visible"
+    :hide-close="hideClose"
+    @close="close"
+  >
+    <v-card-title>
+      {{ $t('home.connect-dialog.connect-title') }}
+    </v-card-title>
+    <div v-if="injectedProvider">
+      <div class="connect__button">
+        <action-button
+          :text="$t('home.connect-dialog.web3-provider')"
+          :enabled="!connecting && !connectingSubkey"
+          :loading="connecting"
+          @click="connect()"
+        />
+      </div>
+      <div class="text-center font-weight-light">
+        {{ $t('home.connect-dialog.divider') }}
+      </div>
+      <i18n
+        v-if="!connectingSubkey"
+        path="home.connect-dialog.raiden-account.description"
+        tag="div"
+        class="connect__raiden-account text-center font-weight-light"
+      >
+        <a v-if="!connecting" @click="connect(true)">
+          {{ $t('home.connect-dialog.raiden-account.link-name') }}
+        </a>
+        <span v-else>
+          {{ $t('home.connect-dialog.raiden-account.link-name') }}
+        </span>
+      </i18n>
```

Comment:
Neglectable: Might make sense to use `<action-button ghost ... />` for the SubKey account options

Reply:
I think I will wait with changing this. Since it is currently just a text link in the middle of a paragraph it might look strange to make it a prominent button. Let's first decide on how to best handle the subkey alternative.